common/src/main/res/values/colors.xml

```diff
@@ -81,6 +81,11 @@
     <color name="text_highlight_blue_bg">#ebf6fd</color>
     <color name="ultra_light_gray">#f5f6f7</color>
     <color name="warning_yellow">#fff9ed</color>
+    <color name="input_error_background">#1AE85C4A</color>
+
+    <!-- Stokes -->
+    <color name="divider_color">@color/gray_50</color>
+    <color name="input_focused_stoke">#cce8fa</color>
```

Comment:
Should this be Stokes or Strokes?  I see that both words have some relation to art.